lib/http/client.cpp

```diff
@@ -3,20 +3,29 @@
 #include "mtxclient/http/client.hpp"
 #include "mtxclient/http/client_impl.hpp"
 
+#if defined(__APPLE__)
+#include <CoreFoundation/CFData.h>
+#include <Security/SecBase.h>
+#include <Security/SecCertificate.h>
+#include <Security/SecPolicy.h>
+#include <Security/SecTrust.h>
+#endif
+
+#include <iostream>
```

Comment:
No need for iostream, use the log function instead.

Reply:
I actually forgot we had one of those in mtxclient =D.  Will update.

Reply:
I mean, in the end it does just std::cerr anyway, but it means in the future we could properly forward the logs to some other log :D